src/redprl/refiner.fun

```diff
@@ -339,21 +340,6 @@ struct
         go
       end
 
-    fun autoSynthesizableNeu sign m =
```

Comment:
excellent

Reply:
Thanks, but you also have to formally approve it if you like it. ;-)